objectivec/Tests/CocoaPods/OSXCocoaPodsTester/OSXCocoaPodsTester.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,290 @@
+// !$*UTF8*$!
```

Comment:
Can you add a README or note somewhere that says that these are generic, Xcode-generated projects without modifications?